.circleci/config.yml

```diff
@@ -81,8 +80,7 @@ jobs:
       - run:
           name: Install dependencies
           command: |
-            pip install --upgrade pip twine wheel
-            pip install --editable .[dev] --upgrade
+            make install
             # pip install --editable git+https://github.com/openfisca/country-template.git@BRANCH_NAME#egg=OpenFisca-Country-Template  # use a specific branch of OpenFisca-Country-Template
```

Comment:
Might as well also move this commented out instruction to Makefile, or delete it altogether.

Reply:
Please don't delete it, the exact command to use a Country Template feature branch in the CI is a hassle to remember, and uncommenting it saves a lot of time and brain power 🙂